### ISSUE TITLE
Another release

### ISSUE THREAD

**Issue by spth** [tenbaht/sduino#104]
It has been over a year since the previous release. At least in SDCC and stm8flash, there were substantial improvements since. It might make sense to release again soon.

**Comment by tenbaht** [comment]
Yes, SDCC 4.0. It is on my todo list. I still need to find a way to streamline the release process. Ideally, use the github actions to automatically pack fresh release files even for the development branch.
